core/arch/arm/include/sm/optee_smc.h

```diff
@@ -290,6 +290,12 @@
 #define OPTEE_SMC_SEC_CAP_HAVE_RESERVED_SHM	(1 << 0)
 /* Secure world can communicate via previously unregistered shared memory */
 #define OPTEE_SMC_SEC_CAP_UNREGISTERED_SHM	(1 << 1)
+/*
```

Comment:
@stuyoder, I'm not using `OPTEE_SMC_SEC_CAP_UNREGISTERED_SHM` at all. I have leaved it as is. I use only `OPTEE_SMC_SEC_CAP_DYNAMIC_SHM` in my patch series.
I decided not to rename `OPTEE_SMC_SEC_CAP_UNREGISTERED_SHM` to `OPTEE_SMC_SEC_CAP_DYNAMIC_SHM` because that can add some confusion. 
At this moment there are at least 30 free bits for capability flags, so I don't see why we need to reuse existing values.